src/components/tokens/TokenMint.js

```diff
@@ -138,6 +138,7 @@ class TokenMint extends React.Component {
               <label className="form-check-label" htmlFor="keepMint">
                 Create another mint output for you?
               </label>
+              <p className="subtitle">Let it marked, unless you know what you are doing</p>
```

Comment:
Leave it selected unless...

Reply:
I think they say "checked" instead of "selected".

Reply:
Updated to 'Leave it checked, unless'